internal/cli/usercmd/push.go

```diff
@@ -237,10 +237,10 @@ func (c *EpinioClient) Push(ctx context.Context, params PushParams) error { // n
 	details.Info("wait for application resources")
 	c.ui.ProgressNote().KeeplineUnder(1).Msg("Creating application resources")
 
-	_, err = c.API.AppRunning(appRef)
-	if err != nil {
-		return errors.Wrap(err, "waiting for app failed")
-	}
+	// _, err = c.API.AppRunning(appRef)
+	// if err != nil {
+	// 	return errors.Wrap(err, "waiting for app failed")
+	// }
 
```

Comment:
```suggestion
```

Reply:
github doesn't let me select the additional empty line that needs to be deleted too :(